include/cutlass/gemm/collective/intel_pvc_mma.hpp

```diff
@@ -187,29 +209,69 @@ struct CollectiveMma<
     static_assert(is_rmem<FrgTensorC>::value, "C tensor must be rmem resident.");
 
     // Tensor to hold input data
-    Tensor tAr = make_tensor<typename TiledMma::ValTypeA>(Shape<Int<get<0>(SubgroupTileShape{}) * FragsK>, _1>{});
-    Tensor tBr = make_tensor<typename TiledMma::ValTypeB>(Shape<Int<get<1>(SubgroupTileShape{}) / 2>, Int<FragsN>>{});
+    constexpr int version =
+        is_same_v<GmemTiledCopyB, XE_2D_U16x16x16x2x1_V> ? 1 : 2;
```

Comment:
what is `version` ? I think it would be better if this information, that is how of B matrix to load goes into the copy traits, rather than having a constexpr if here

Reply:
We are working on the full-feature of cute. If possible, we can solve it in the furture PRs when cute's full-feature is ready to go.